data/human/culture conversations.txt

```diff
@@ -1101,3 +1101,279 @@ mission "Little Vienna in New Austria"
 			label backtoship
 			`	With the atmosphere steeped in nostalgia for a distant city on a distant world long past, you return to your ship.`
 
+phrase "racetrack"
+	word
+		"Racetrack"
+		"Speedway"
+		"Arena"
+		"Course"
+		"Circuit"
+		"Glideway"
+		"Hover Circuit"
+		"Velocity Course"
+		"Hoverloop"
+
+phrase "sunracer opening"
+	word
+		"As you walk through the din of Sunracer's bustling spaceport"
+		"As you wade through the clang and clatter of Sunracer's ever-churning shipyards"
+		"Amidst the city's persisting glitz"
+		"Lost in a maze of hangars and half-built hulls"
+		"Navigating through the organized chaos of Megaparsec shipyards"
+
+phrase "sunracer hovercraft adhook"
+	word
+		"dazzling holographic display"
+		"radiant, floating billboard"
+		"vivid holographic advertisement"
+	word
+		" seizes your attention"
+		" catches your eye"
+		" appears overheard"
+
+phrase "sunracer hovercraft adcloser"
+	word
+		"The ad flashes sleek"
+		"Photo-realistic images show agile"
+		"The visuals display streamlined"
+	word
+		" hovercraft "
+		" vehicles "
+	word
+		"slicing through a twisting course in untamed terrain, "
+		"racing along a serpentine, rugged track, "
+		"carving through wild, winding roads, "
+	word
+		"with the ad copy "
+		"the display's text "
+		"the words below "
+	word
+		"promising high-speed thrills for the audience"
+		"teasing a spectacle of breakneck action"
+		"offering heart-pounding, high-octane excitement"
+		"beckoning wealthy corporate sponsors to back the adrenaline-fueled event"
+		"offering the perk of free parking"
+		"simply stating 'Sponsored by Megaparsec'"
+		
+phrase "sunracer city to wilderness description"
+	word
+		"Sunracer's bustling shipyards"
+		"the vibrant Megaparsec metropolis"
+		"the clamor of Sunracer's industrial hub"
+	word
+		" fades into "
+		" gives way to "
+		" transitions to "
+	word
+		"shimmering lakes dotted with sailboats and fishing vessels"
+		"picturesque rolling hills of sprawling farmlands"
+		"a serene stretch of raw, unspoiled nature"
+
+phrase "sunracer fan tshirt"
+	word
+		"clad in Megaparsec Racing Technologies gear, with a large Quicksilver emblazoned across the front of his shirt"
+		"wearing Megaparsec Racing Technologies apparel - his shirt covered with hundreds of tiny Splinter, Manta, and Quicksilver silhouettes"
+		"dressed in a Megaparsec Racing Technologies sweater that has the words 'Mighty fast, impossibly light - Megaparsec' at its center"
+		"dressed in a Syndicate Performance Racing shirt with the centered words 'Smart racers know race pace is key. Smart racers race for Syndicate Performance Racing"
+		"wearing a Deep Sky Dynamics hat with the words 'Peace, Prosperity, Progress... Podium'"
+		"dressed in a Delta V Grand Prix sweater, the words 'The Wild Pace of Plasma' surrounded by various images of tiny chipmunks, greyhounds, capybaras, and impalas running across the shirt, leaving tiny footprints"
```

Comment:
```suggestion
		`dressed in a Syndicate Performance Racing shirt with the centered words "Smart racers know race pace is key. Smart racers race for Syndicate Performance Racing"`
		`wearing a Deep Sky Dynamics hat with the words "Peace, Prosperity, Progress... Podium"`
		`dressed in a Delta V Grand Prix sweater, the words "The Wild Pace of Plasma" surrounded by various images of tiny chipmunks, greyhounds, capybaras, and impalas running across the shirt, leaving tiny footprints`
```

"Impalas" also seems deeply wrong to me, but it is grammatically correct (and more common than "impala"?) so it's probably good as-is.